### DISCUSSION TITLE
Is there a way to disable tenant scoping for pages?

### DISCUSSION THREAD

**Discussion by boris-glumpler** [filamentphp/filament#9707] [Help]
I know I can disable tenant scoping for resources by setting `$isScopedToTenant` to `false`. There doesn't seem to be something similar for pages. My use case here is for a `confirm password` page to use in the `password.confirm` middleware. I can pass a custom route to that middleware, but can't pass any route parameters along with it. Any page added to a tenant-enabled panel expects a `tenant` parameter, tho. Does anybody know of a solution here? Cheers!

**Comment by krzysztofengineer** [comment]
any updates on this?

**Comment by kraziegent** [comment]
I'll like to know how this can be achieved too. Want users to have limited functionality if they are not part of a team.
